README.md

```diff
@@ -303,9 +303,9 @@ python3 setup.py develop
 
 If you want to build legacy python code, please refer to [Building on legacy code and CUDA](https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md#building-on-legacy-code-and-cuda)
 
-**CPU-only builds**
+**CPU-only or Intel GPU builds**
```

Comment:
What's the behavior if users install CUDA SDK? Actually, it is sort of confusing.

Reply:
Hi Eikan, do you suggest to add a note telling the users that they should not install cuda sdk if they are building PyTorch on a machine with Intel GPU?

Reply:
I'm not sure if we should highlight it since it is CPU-only.